libs/dh/dev-examples/feature-processes/src/table.component.ts

```diff
@@ -54,7 +58,12 @@ import { DhProcessStateBadge } from '@energinet-datahub/dh/wholesale/shared';
     >
       <h3>{{ t('headline') }}</h3>
 
-      <watt-table [dataSource]="dataSource" [columns]="columns" [loading]="dataSource.loading">
+      <watt-table
+        [dataSource]="dataSource"
+        [columns]="columns"
+        [loading]="dataSource.loading"
+        (rowClick)="navigate($event.id)"
```

Comment:
suggestion: we could inline here as well (if we think that is best practice, not sure)

Reply:
I agree a single line is good to inline fixed in https://github.com/Energinet-DataHub/greenforce-frontend/pull/3958/commits/20e4154dee826ad501630d27de7f5d1c09a04d6e